API.md

```diff
@@ -397,7 +397,7 @@ Emulating typing in an input using `typeIn`
 
 
 ```javascript
-typeIn('hello world');
+typeIn('input', 'hello world');
```

Comment:
the `API.md` file is automatically generated. you will also have to change this in the corresponding code comment :)

Reply:
🤦